src/core/udb-api.service.js

```diff
@@ -506,6 +506,9 @@ function UdbApi(
           .get(appConfig.baseUrl + 'user/permissions/', defaultApiConfig)
           .success(storeAndResolvePermissions)
           .error(function (error) {
+            if (!error) {
```

Comment:
Nit: the first argument for `.error()` is actually `data`. That might make this clearer than `error` because it's weird that `error()` would be called without an `error` 😛 

https://code.angularjs.org/1.0.8/docs/api/ng.$http

Reply:
There's also a second argument `status`, which might might also be a good (better?) check than `data`

Reply:
Yes I'll look into it, strange that the first arg is data